units/getty@.service.in

```diff
@@ -56,6 +56,7 @@ ImportCredential=tty.virtual.%I.agetty.*:agetty.
 ImportCredential=tty.virtual.%I.login.*:login.
 ImportCredential=agetty.*
 ImportCredential=login.*
+ImportCredential=shell.*
```

Comment:
Hmm, are you sure this works? Since #33996 we no longer ask `login` to propagate down arbitrary envvars. There's a proposal at util-linux currently for allowing to customize this.